docs/libxmp.rst

```diff
@@ -531,7 +531,7 @@ void xmp_scan_module(xmp_context c)
   performed by `xmp_load_module()`_ and this function should be called only
   if `xmp_set_player()`_ is used to change player timing (with parameter
   ``XMP_PLAYER_VBLANK``) in libxmp 4.0.2 or older, or if
-  `xmp_set_tempo_factor`_ is used to change the base tempo factor.
+  ``xmp_set_tempo_factor`` is used to change the base tempo factor.
```

Comment:
I was trying to make API links (like line 532) but left off the `()` I think...

Reply:
I see - updated.